### ISSUE TITLE
Web page is not being navigated sequentially with the Tab key at Home screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#823]
**Describe the bug**
Web page is not being navigated sequentially with the Tab key at Home screen. The sequence of focus is unordered.

**To Reproduce**
1. Go to Home screen (http://localhost:9992/cm/app/?view=home&pathType=unknown)
2. Use Tab key to focus on different components of the screen.
3. Focus is not moving in sequence and is not going to "Add New", "Search", "Browse Library" buttons, etc.

**Expected behavior**
Web page should be navigated sequentially with the Tab key.

**Additional context**
Navigate all the options available for similar issue and fix.
